pages/landing/hipaa/template/pricing/index.html

```diff
@@ -18,31 +18,31 @@
     description:
       Early access subscribers will receive a CIS staging environment when invited<br>
       and a fully HIPAA-compliant production infrastructure when it’s ready.
-    price: 3060
-    strike_price: 10000
-    flat_fee: 7425
-    strike_flat_fee: 9900
+    price: "3,060"
+    strike_price: "10,000"
+    flat_fee: "7,425"
+    strike_flat_fee: "9,900"
     image: /assets/img/logos-compliance/hipaa.png
     image_styles: "max-width: 130px"
     features:
-      - name: HIPAA Compliant Reference Architecture
-        description: Audited by 3rd party auditors to insure compliance.
-      - name: HIPAA Certified Sample Application
-        description: A world-class sample application you can easily extend.
-      - name: CIS Reference Architecture Immediately
-        description: Deploy dev and stage infrastructure in environments that approach HIPAA compliance now.
+      - name: CIS-compliant today, HIPAA-compliant soon
+        description: Deploy dev and stage infrastructure today and start building your application. HIPAA compliance coming soon!
+      - name: HIPAA-compliant reference architecture
+        description: Assessed by qualified 3rd parties for peace of mind.
+      - name: HIPAA-compliant sample applications
+        description: World-class sample applications that you can easily extend.
       - name: Infrastructure as Code Library
-        description: Access our entire library to augment and extend your Reference Architecture.
-      - name: Gruntwork Customer Support
-        description: Get help when you need help.
-      - name: Gruntwork Training Libraries
+        description: Access our entire library to augment and extend your reference architecture.
+      - name: Customer support
+        description: Get help when you need it.
+      - name: Training library
```

Comment:
Will a customer who has only seen these landing pages have any idea what this is?

Reply:
Updated in 83985751.